### PR TITLE
Feature/frequent positions

### PR DESCRIPTION
**Pull request by mjoyce91**
Merge https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/1961 first

- Adds Frequent Positions component to research pane
- Not connected to real data yet, using static data
- "Plus" button passes up the position object, but otherwise does nothing

Screen Shot 2022-02-04 at 3 31 24 PM

### CODE REVIEW

File: src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx [hunk 1]
```diff
@@ -6,26 +6,58 @@ import MediaQuery from 'Components/MediaQuery';
...
+    position_number: '0000000000',
+    position_title: 'ECONOMIC OFFICER LONGER NAME HERE (' },
+  { org: 'A',
+    position_number: '0000100000',
+    position_title: 'ECONOMIC OFFICER' },
+  { org: 'ORG NAME',
+    position_number: '0000000000',
+    position_title: 'TRAINING' },
+];
+positions = [...positions, ...positions, ...positions, ...positions];
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx:26]: ❤️

File: src/sass/_agendaItemMaintenance.scss [hunk 2]
```diff
@@ -68,4 +68,101 @@
...
+    th {
+      background-color: $dos-blue;
+      border: 0;
+      color: $color-white;
+      font-weight: bold;
+    }
+
+    #customers tr:nth-child(even){background-color: #f2f2f2;}
+
+    #customers tr:hover {background-color: #ddd;}
```
Comment (elizabeth-jimenez) [inline src/sass/_agendaItemMaintenance.scss:165]: 🧼  hehe

**Review by elizabeth-jimenez — APPROVED** [review]
@mjoyce91 Just a bit of cleanup, but besides that v cool

**Comment by mjoyce91** [comment]
Need to fix conflicts

File: src/sass/_agendaItemMaintenance.scss [hunk 3]
```diff
@@ -86,4 +86,75 @@
...
+    }
+    tr > *:nth-child(3) {
+      flex: 6
+    }
+    tr > *:nth-child(4) {
+      flex: 12
+    }
+
+    tbody > tr {
+      border: 2px solid $bg-gray-dark-1;
```
Comment (elizabeth-jimenez) [inline src/sass/_agendaItemMaintenance.scss:120]: Discussed in Slack: maybe making 1px

Comment (mjoyce91) [reply src/sass/_agendaItemMaintenance.scss:120]: Merged before I saw this

Comment (elizabeth-jimenez) [reply src/sass/_agendaItemMaintenance.scss:120]: hmm 🤔  suspicious ;p

**Pull request merged**